app/views/posts/index.html.erb

```diff
@@ -1 +1,30 @@
-<h1>Here is a list of posts for a given user</h1>
+<html>
+  <head>
+    <%= stylesheet_link_tag "styles" %>
+  </head>
+
+  <body>
+    <div class='container'>
+      <div class='user-card'>
+        <%= render partial: 'shared/user_card', locals: { user: @user } %>
+      </div>
+
+      <% @user.posts.each do |post| %>
+          <a href="<%= user_post_path(@user, post) %>">
+            <div class='post-container'>
+              <h2 class='post-title'><%= post.title %></h2>
+              <p><%= truncate(post.text, length: 50, separator: ' ') %></p>
+              <p class='counters'>Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %></p>
+            </div>
+            
+            <% if post.comments.present? %>
+              <div class='comments-section'>
+                <% post.comments.each do |comment| %>
```

Comment:
- [ ] Looking at the wireframe just the first five recent comments are displayed. You should therefore be using the recent_comments method (`<% post.recent_comments.each do |comment| %>`) found in the post model that you created in previous milestone.
```
  def recent_comments
    comments.order(created_at: :desc).limit(5)
  end
```
**See screenshot of `app/views/posts/index.html.erb` page below**
![Screen Shot 2023-09-01 at 6 47 28 AM](https://github.com/ClintonYounge/My-Blog-App/assets/62849872/36c76176-8dad-437a-96eb-cb4713810e12)

Reply:
Thanks. Changed.